assembl/graphql/schema.py

```diff
@@ -472,6 +472,7 @@ class Mutations(graphene.ObjectType):
     create_thematic = CreateThematic.Field(description=docs.CreateThematic.__doc__)
     update_thematic = UpdateThematic.Field(description=docs.UpdateThematic.__doc__)
     delete_thematic = DeleteThematic.Field(description=docs.DeleteThematic.__doc__)
+    update_ideas = UpdateIdeas.Field(description=docs.UpdateIdeas.__doc__)
```

Comment:
Was about to add a comment but I guess that you don't remove `create_idea`, etc because front-end code has not been updated yet.

Reply:
yes